operations/iuf/IUF.md

```diff
@@ -402,18 +409,20 @@ These [examples](examples/iuf_resume.md) highlight common use cases of `iuf resu
 
 #### `restart`
 
-The `restart` subcommand is specified by the administrator to restart a previously aborted or failed IUF session for a given activity. Specifically, it re-executes the most recent IUF session executed via `iuf run`. All stages
-of the IUF session will be re-executed, regardless of whether they succeeded or failed during the previous invocation of `iuf run`.
+The `restart` subcommand is specified by the administrator to restart a previously aborted or failed IUF session for a given activity. Specifically, it re-executes the most recent IUF session executed via `iuf run`. Any Argo step
```

Comment:
Suggested update:

Run the `restart` subcommand to restart a previously aborted or failed IUF session. This re-executes the most recent IUF session executed via `iuf run`. Any Argo step that already executed successfully is skipped if possible; the Argo UI displays the step, but the corresponding log file will contain a message if the step operations were skipped. If the `-f`
argument is specified, all stages specified by the most recent `iuf run` will be re-executed, regardless of whether they succeeded or failed during the previous invocation of `iuf run`.

Reply:
Fixed in next commit.